.azure-pipelines/util/analyze-steps.yml

```
@@ -31,6 +31,16 @@ steps:
 
 - pwsh: 'Install-Module "platyPS", "PSScriptAnalyzer" -Force -Confirm:$false -Scope CurrentUser'
   displayName: 'Install PowerShell Dependencies'
+
+- task: PowerShell@2
+  displayName: 'Install latest modules'
+  inputs:
+    targetType: 'inline'
+    script: |
+      New-Item -ItemType Directory -Path "Az-Cmdlets-latest"
+      Invoke-WebRequest -Uri "https://azpspackage.blob.core.windows.net/release/Az-Cmdlets-latest.tar.gz" -OutFile "Az-Cmdlets-latest/Az-Cmdlets-latest.tar.gz" -MaximumRetryCount 2 -RetryIntervalSec 1
+      tar -xvzf "Az-Cmdlets-latest/Az-Cmdlets-latest.tar.gz" -C "Az-Cmdlets-latest"
+      . Az-Cmdlets-latest/InstallModule.ps1
```

```suggestion
      . Az-Cmdlets-latest/InstallModule.ps1
    pwsh: true
```
add pwsh: true to specify to use powershell 7 (instead of windows powershell)